CMakeSettings.json

```diff
@@ -0,0 +1,162 @@
+﻿{
+  "configurations": [
+    {
+      "name": "Windows_NT.arm64.Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "buildRoot": "${projectDir}\\artifacts\\obj\\Windows_NT.arm64.Debug\\crossgen",
+      "installRoot": "${projectDir}\\artifacts\\bin\\Windows_NT.arm64.Debug",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "inheritEnvironments": [ "msvc_arm64" ],
+      "variables": [
+        {
+          "name": "CLR_CMAKE_HOST_ARCH",
+          "value": "x64",
+          "type": "STRING"
+        },
+        {
+          "name": "CLR_CMAKE_TARGET_ARCH",
+          "value": "arm64",
+          "type": "STRING"
+        },
+        {
+          "name": "CLR_MANAGED_BINARY_DIR",
+          "value": "${projectDir}\\artifacts\\bin",
+          "type": "FILEPATH"
+        }
+      ]
+    },
+    {
+      "name": "Windows_NT.arm64.Release",
+      "generator": "Ninja",
+      "configurationType": "Release",
+      "buildRoot": "${projectDir}\\artifacts\\obj\\Windows_NT.arm64.Release\\crossgen",
+      "installRoot": "${projectDir}\\artifacts\\bin\\Windows_NT.arm64.Release",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "inheritEnvironments": [ "msvc_arm64" ],
+      "variables": [
+        {
+          "name": "CLR_CMAKE_HOST_ARCH",
+          "value": "x64",
+          "type": "STRING"
+        },
+        {
+          "name": "CLR_CMAKE_TARGET_ARCH",
+          "value": "arm64",
+          "type": "STRING"
+        },
+        {
+          "name": "CLR_MANAGED_BINARY_DIR",
+          "value": "${projectDir}\\artifacts\\bin",
+          "type": "FILEPATH"
+        }
+      ]
+    },
+    {
+      "name": "Windows_NT.x64.Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64" ],
```

Comment:
should we be using msvc_x64_x64 instead?

Reply:
I looked at the description for these and the difference is that the current one "Target x64 Windows with the MSVC compiler" whereas the one with the extra _x64 "Target x64 Windows with the 64-bit MSVC compiler". I don't think we need to be that specific about the compiler bitness.